### PR TITLE
Likegen

### PR DESCRIPTION
**Pull request by njc3006**
Like Generation, and user preferred age min and max +-10 and modified generation methods to include the csv parse users.

### CODE REVIEW

File: src/helpers/UserGen.java [hunk 1]
```diff
@@ -240,6 +236,14 @@ public UserGen() {
             gc.set(Calendar.DAY_OF_YEAR, dayOfYear);
             dob = new Date(gc.getTime().getTime());
 
+            Integer age = Period.between(dob.toLocalDate(), LocalDate.now()).getYears();
```
Comment (jarrydlee) [inline src/helpers/UserGen.java:239]: Can this not mean your age could be almost 0?

**Comment by jarrydlee** [comment]
Good stuff making that date helper. Apart from that one comment I like it

File: src/helpers/UserGen.java (hunk 1 above)
Comment (njc3006) [inline src/helpers/UserGen.java:239]: What? The dob will be between 1950 and 1998, so age is always greater than 16

Comment (jarrydlee) [inline src/helpers/UserGen.java:239]: Between dob and localdate will be dob and this year won't it? Am I missing something here?

Comment (jarrydlee) [inline src/helpers/UserGen.java:239]: Yep read the docs wrong, this is good then

**Pull request merged**